www/index.html

```diff
@@ -487,10 +491,17 @@ <h2 data-i18n="configure-title">Configuration</h2>
                             <br /> Scanning for archives... Please wait <img src="img/spinner.gif" alt="Please wait..." />
                         </div>
                         <div id="chooseArchiveFromLocalStorage" style="display: none;">
-                            <br /> Please select the archive you want to use : <select id="archiveList"
-                                class="form-control"></select>
-                            <br /><button type="button" class="btn btn-light" id="btnRescanDeviceStorage">Rescan</button>
-                            Rescans your SD Cards and internal memory
+                            <div style="float: left;display: none;" id="fileCountDisplay">
+                                <p id="numberOfFilesDisplay" style="float: left;">0</p>
+                                <span style="float: left;">&nbsp;archives found in selected location.&nbsp;</span>
```

Comment:
Two lines above don't make much sense. It's all one paragraph, so should all be wrapped in a single pair of `<p>...</p>` tags. The `numberOfFilesToDisplay` element should be a `<span>`, not a `<p>`, and no need for span tags around the "archives found in selected location" element, since you won't be making programmatic substitutions there at all (and it certainly shouldn't be floated left!).